src/Command/DoctrineDecryptDatabaseCommand.php

```diff
@@ -94,65 +84,78 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         // Start decrypting database
+        $_ENV['DOCTRINE_SKIP_ENCRYPT'] = true;
         $output->writeln(''.PHP_EOL.'Decrypting all fields. This can take up to several minutes depending on the database size.');
 
-        $valueCounter = 0;
+        $platform = new MySQL80Platform();
```

Comment:
I would not hard-code this, but fetch it from the actual connection:

```suggestion
        $platform = $this->entityManager->getConnection()->getDatabasePlatform();
```